internal/dbmodel/model.go

```diff
@@ -115,10 +84,7 @@ func (m *Model) SetOwner(u *Identity) {
 // will need to be filled in manually by the caller of this function.
 func (m *Model) FromJujuModelInfo(info jujuparams.ModelInfo) error {
```

Comment:
do we still need this method?

Reply:
and other methods that transform to/from juju types..

Reply:
we still need them because we are still merging info coming from the controller with the info that we have.
For example:
we take info coming from controllers, we change the OwnerIdentity, CloudCredentials. 
And then we return this to the client.